syz-cluster/pkg/report/testdata/1.out.txt

```diff
@@ -0,0 +1,44 @@
+syzbot has found the following issue(s) in the series
```

Comment:
@ramosian-glider @melver please also take a look, this is what we are going to send to kernel mailing lists

Reply:
It's short and crisp which is good.

Only thing that might be worthwhile is more clearly spelling out how to reproduce the issues, because I think that's going to be the most effective thing a developer can do to check their work. It's likely too inefficient to wait for syzbot to retest (if that's even planned) changes before it's sent as a v+1.

I see links to config, and mention of arch and compiler, but besides that it's unclear how I'd go about trying to reproduce the problem.